README.md

```diff
@@ -83,15 +83,17 @@ A key design principle for us is composability as in any new dtype or layout we
 ### Installation
 `torchao` makes liberal use of several new features in Pytorch, it's recommended to use it with the current nightly or latest stable version of PyTorch.
 
-Stable release from Pypi which will default to PyTorch 2.3.1 and CUDA 12.1
+Install the torch version you want from https://pytorch.org/
```

Comment:
will any torch version work? The previous line suggests current nightly or latest stable. Can we add that explicitly here (and maybe remove that line)

Reply:
made this clearer now